### PR TITLE
Google sheets premade integration

### PR DESCRIPTION
**Pull request by vicmgs**
working google form id to test creation with: 1FAIpQLSeA8B0QoGhFurLrtiCMp55QAtQFP502C_NDvOTLGV9kTZVZaQ

### CODE REVIEW

File: assets/js/util/integrationInfo.js [hunk 1]
```diff
@@ -43,6 +43,13 @@ export const PREMADE_CHANNEL_TYPES = [
     info: "Cargo is an in-house mapping tool used at Helium.",
     docLink: "https://docs.helium.com/use-the-network/console/integrations/cargo"
   },
+  {
+    name: "Google Sheets",
+    link: "/integrations/new/googlesheet",
+    img: `${Google}` ,
+    info: "This Integration streamlines sending data to a Google Sheet by using a Google Form.",
+    docLink: "https://docs.helium.com/use-the-network/console/integrations/tago"
```
Comment (ambuznego) [inline assets/js/util/integrationInfo.js:51]: can we use the correct placeholder so we don't gotta change this later

File: assets/js/components/channels/forms/GoogleSheetForm.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+      this.props.updateGoogleFieldsMapping(fieldsMapping)
+    }
+  }
+
+  render() {
+    return(
+      <div>
+        <div>
+          <Text style={{ display: 'block' }}>Enter Google Form Id:</Text>
```
Comment (ambuznego) [inline assets/js/components/channels/forms/GoogleSheetForm.jsx:83]: `Id` -> `ID` 👼

File: assets/js/util/integrationInfo.js (hunk 1 above)
Comment (vicmgs) [reply assets/js/util/integrationInfo.js:51]: yisss

File: assets/js/components/channels/ChannelNew.jsx [hunk 3]
```diff
@@ -32,6 +38,50 @@ const { Option } = Select
...
+  Object.entries(definition).forEach(
+    ([fieldName, entry]) => {
+      const name = slugify(fieldName);
+      fields.push(`"${name}": "${entry}"`);
+      payloads.push(`"${name}": "FILL ME IN"`);
+    }
+  )
+
+  return `function Decoder(bytes, port) {
+// TODO: Transform bytes to decoded payload below
```
Comment (ambuznego) [inline assets/js/components/channels/ChannelNew.jsx:55]: shouldn't this and line 59 be indented

**Review by ambuznego — APPROVED** [review]

**Pull request merged**